Katello/katello
# Fixes #35707 - Add action link when there are 0 installable errata

#### What are the changes introduced in this pull request?

new host detail page - When a host has applicable errata but 0 installable errata, this adds the following:

1. Text changes as outlined in the Redmine
2. Add action link to "View applicable errata" which changes the toggle.

![image](https://user-images.githubusercontent.com/22042343/208138912-51da8fcf-582f-4626-8886-e7ff3daec88d.png)


#### Considerations taken when implementing this change?

Secondary actions in `EmptyStateMessage.js` were conditionally either an `<a>` or a link-styled button. I renamed some things to make it a bit more clear.

#### What are the testing steps for this pull request?

1. Publish a content view with Sea Erratum (sync http://jlsherrill.fedorapeople.org/fake-repos/needed-errata/)
2. Promote this CV to a lifecycle environment
4. Assign your host to this CV and environment
5. Create an errata exclude filter and exclude Sea Erratum
6. Publish a new CV version to that same LCE
7. You should now have 1 applicable and 0 installable errata on the host. View the Content > Errata tab on the new host detail page.
